### PR TITLE
DEV-4898: map-makers audit and readme

### PR DESCRIPTION
**Pull request by MSilva95**
fix readme and audit
add prints

### CODE REVIEW

File: subjects/mobile-dev/map-markers/README.md [hunk 1]
```diff
@@ -1,38 +1,67 @@
...
+In this project, your team will create an app that uses the `Google Maps API`. The app must allow users to save and display their favorite places on a map.
 
-Create an app where you can save the list of your favorite places and show them in the app.
+### Instructions
 
-In this raid, your team should implement an app which uses google maps.
+The app should have the following functionality:
 
-### Objective
+A `TabBar` with three screens:
```
Comment (nprimo) [inline subjects/mobile-dev/map-markers/README.md:9]: In the audit, it is asked to look for a `Bottom TabBar` while here is "simply" `TapBar`

File: subjects/mobile-dev/map-markers/audit/README.md [hunk 2]
```diff
@@ -1,35 +1,37 @@
 #### Functional
 
-> In order to run and hot reload app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
+> In order to run and hot reload the app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
 
-###### Does the application run without crashes?
+###### Does the app run without crashing?
 
-###### Does the application have Bottom tabbar?
+###### Does the app have `Bottom TabBar`?
```
Comment (nprimo) [inline subjects/mobile-dev/map-markers/audit/README.md:7]: In the subject is requested to have a `TapBar`

File: subjects/mobile-dev/map-markers/README.md [hunk 3]
```diff
@@ -1,38 +1,67 @@
...
 
-Implement following functionality:
+- A `Maps` screen like [google maps](https://codelabs.developers.google.com/codelabs/google-maps-in-flutter#0) where the user can see their favorite places marked on the map with an `info view`.
 
-- Tabbar app with 3 screens:
-  - Google maps screen [example](https://codelabs.developers.google.com/codelabs/google-maps-in-flutter#0)
-  - List of your favorite places.
-  - Info about your app. Names of the authors, their email adresses, and general description of what your application does.
+- A `favorite places` screen which will have a list of the user's favorite places.
```
Comment (nprimo) [inline subjects/mobile-dev/map-markers/README.md:13]: Minor formatting differences compared to the audit (`Favorite Places` in the audit)

File: subjects/mobile-dev/map-markers/audit/README.md [hunk 4]
```diff
@@ -1,35 +1,37 @@
...
+###### Does the app show the list of favorite places after closing and reopening the app?
 
-###### Does the application contain an information page?
+###### Can you delete a favorite place from your list?
 
-###### Does map show markers with favorite places?
+###### If the favorite place is deleted, does the app remove it from the map?
 
-###### When map is tapped, does dialog window show 2 textfields: name and description?
+###### Does the app contain an information page on the `Info` screen?
```
Comment (nprimo) [inline subjects/mobile-dev/map-markers/audit/README.md:19]: Small differences between name of the page asked to be created in the subject (`InfoPage`)

**Review by nprimo — CHANGES_REQUESTED** [review]
There are some minor differences between `audit` and `subject` names

File: subjects/mobile-dev/map-markers/README.md (hunk 1 above)
Comment (MSilva95) [reply subjects/mobile-dev/map-markers/README.md:9]: done

File: subjects/mobile-dev/map-markers/audit/README.md (hunk 2 above)
Comment (MSilva95) [reply subjects/mobile-dev/map-markers/audit/README.md]: done

File: subjects/mobile-dev/map-markers/README.md (hunk 3 above)
Comment (MSilva95) [reply subjects/mobile-dev/map-markers/README.md]: done

**Pull request merged**